developers.diem.com/src/README.md

```diff
@@ -45,8 +45,8 @@ unzip diem.zip
 ## Publishing
 
 The site is hosted on GitHub pages, via the `gh-pages` branch of the `website'
-[GitHub repo](https://github.com/libra/website).
+[GitHub repo](https://github.com/diem/website).
```

Comment:
None of this publishing stuff is true any longer. I think we need a vet of the READMEs and other info files for accuracy. We can leave for now and do a cleanup pass later.